homeassistant/components/websocket_api/const.py

```diff
@@ -23,6 +23,7 @@
 ERR_ID_REUSE = "id_reuse"
 ERR_INVALID_FORMAT = "invalid_format"
 ERR_NOT_FOUND = "not_found"
+ERR_NOT_LOADED = "not_loaded"
```

Comment:
No need to make this generic since it's only used inside the zwave_js integration. This list is not exclusive, you can make any error code.

Reply:
Ok. So far it seemed no custom codes where used, but I'll move it.

Reply:
Moved it: https://github.com/home-assistant/core/pull/50188